provision/common/Taskfile.yaml

```diff
@@ -240,3 +242,28 @@ tasks:
       - .task/subtask-{{.TASK}}.yaml
     generates:
       - ../tlsdisableagent/tlscheckdisable-agent.jar
+
+  clone-keycloak-grafana-dashboards:
+    desc: "Clone the Keycloak grafana repository"
+    internal: true
+    dir: ".task"
+    cmds:
+      - rm -rf "keycloak-grafana-dashboard" || true
+      - git clone --branch "{{.KC_GRAFANA_DASHBOARDS_BRANCH}}" "{{.KC_GRAFANA_DASHBOARDS_REPOSITORY}}" "keycloak-grafana-dashboard" --depth 1
+      - git -C "keycloak-grafana-dashboard" checkout "{{.KC_GRAFANA_DASHBOARDS_BRANCH}}"
+      - git -C "keycloak-grafana-dashboard" pull
+      - |
+        cd keycloak-grafana-dashboard/dashboards
+        for F in *; do
+          cp ${F} copy.json;
+          DS_PROMETHEUS=PBFA97CFB590B2093 envsubst '${DS_PROMETHEUS}' < copy.json > ${F};
```

Comment:
This is not very nice but I haven't found how to import dashboards with `__inputs` with Grafana operator :/. 

The suggestion I found was to not use `__inputs` but as far as I understand it is necessary for manually importing dashboards using UI and also when uploading dashboards to Grafana labs (if we decide to do that later).

Reply:
Also, we may want to document this in Grafana guide for people that want to automate it. I created a tracker in our board.